splinepy/multipatch.py

```diff
@@ -270,10 +332,10 @@ def determine_interfaces(self):
 
         return self.interfaces
 
-    def add_boundary_with_function(
+    def boundary_from_function(
```

Comment:
```suggestion
    def boundaries_from_function(
```
or you can change all the other one to `boundary_*`

Reply:
The others auto assign multiple boundaries, this one does one at a time. I reconsidered this change and changed it back, :sweat_smile: